### PR TITLE
Remote2

### PR DESCRIPTION
**Pull request by GraceFu**
- Fix host field of author object
- Delete irrelevant fields in author profile (frontend)

### CODE REVIEW

**Review by zhininghjl — CHANGES_REQUESTED** [review]
should this be the GitHub URL rather than the name?

File: api/templates/profile.html [hunk 1]
```diff
@@ -101,39 +101,15 @@ <h1>{{ author.displayName }}</h1>
...
-                                <input type="text" class="form-control" value="(320) 380-4539">
-                            </div>
-                        </div>
-                        <div class="row mb-3">
-                            <div class="col-sm-3">
-                                <h6 class="mb-0">Address</h6>
-                            </div>
-                            <div class="col-sm-9 text-secondary">
-                                <input type="text" class="form-control" value="Bay Area, San Francisco, CA">
+                                <input type="text" class="form-control" value="{{ author.displayName }}">
```
Comment (zhininghjl) [inline api/templates/profile.html:112]: should this be github url rather than name?

File: api/views/auth.py [hunk 2]
```diff
@@ -18,6 +18,7 @@ def signup_request(request):
             author.github = form.cleaned_data["github"]
             author.displayName = form.cleaned_data["displayName"]
             author.url = "https://" + request.get_host() + "/api/author/" + author.authorID
+            author.host = request.get_host()
```
Comment (zhininghjl) [inline api/views/auth.py:21]: I think the host should have `https://` as the prefix

Comment (GraceFu) [reply api/views/auth.py]: fixed!

File: api/templates/profile.html (hunk 1 above)
Comment (GraceFu) [reply api/templates/profile.html]: fixed

**Pull request merged**